### ISSUE TITLE
DTS ISO should contain all recent Dasharo Community Support binaries, hashes and signatures

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#457]
### The problem you're addressing (if any)

As explained in #456, there are various users in the Dasharo community, some would like to flash Dasharo without network connectivity which means Dasharo binaries (hash and signature) should already be in DTS ISO.

### Describe the solution you'd like

Add the most recent binaries to the DTS ISO image.

### Where is the value to a user, and who might that user be?

No need for a network connection.

**Comment by macpijan** [comment]
I propose to drop support for ISO image type, at least until it works and is documented: what works on ISO, what does not.

Currently, ISO is basically unusable (due to being read only and various DTS actions writing to disk), and it's main feature is to trigger support requests in Dasharo Matrix channels.

**Comment by zirblazer** [comment]
> I propose to drop support for ISO image type, at least until it works and is documented: what works on ISO, what does not.
> 
> Currently, ISO is basically unusable (due to being read only and various DTS actions writing to disk), and it's main feature is to trigger support requests in Dasharo Matrix channels.

I agree with this. I was the one that requested DTS to be available in ISO format so that you can use it with mainstream multiboot tools like Ventoy who have no idea what the other WIC image is, but after you began to provide the ISOs, in practice they were effectively useless as the DTS read-write dependency became apparent (Plus I recall at least a few cases were dding the ISO still didn't worked whereas the WIC image did). And since this was never fixed, there is no reason for an ISO that never worked to exists on the first place. Which doesn't mean that I believe that delivering DTS in ISO was a bad idea itself, but it required complementary work to provide a *working* DTS ISO.

Note that I recall that there was [at least a person](https://github.com/Dasharo/dasharo-issues/issues/288#issuecomment-1755271518) that considered using the [Ventoy persistence plugin](https://www.ventoy.net/en/plugin_persistence.html), so the read-write dependency may still be possible to be fixed Ventoy side. But the instructions are a bit... complicated for my taste. I have no idea whenever it supports a more generic form of CoW layered over the ISO.

**Comment by macpijan** [comment]
The ISO support is dropped.
We do not intend to include binaries in the image right now either.
